packages/playground/src/components/select_node.vue

```diff
@@ -188,43 +212,115 @@ watch(
 );
 
 watch(
-  () => ({ ...props.filters }),
-  (value, oldValue) => {
+  () => props.filters,
+  async (value, oldValue) => {
     if (
       value.farmId === oldValue.farmId &&
-      value.diskSizes === oldValue.diskSizes &&
+      value.diskSizes.length === oldValue.diskSizes.length &&
+      new Set([...value.diskSizes, ...oldValue.diskSizes]).size === value.diskSizes.length &&
       value.cpu === oldValue.cpu &&
       value.memory === oldValue.memory &&
       value.certified === oldValue.certified &&
       value.rentedBy === oldValue.rentedBy &&
       value.hasGPU === oldValue.hasGPU
     )
       return;
-    shouldBeUpdated.value = true;
+    if (props.selection === "manual" && selectedNode.value) {
+      await validateManualSelectedNode(selectedNode.value);
+    } else {
+      shouldBeUpdated.value = true;
+    }
+  },
+  { deep: true },
+);
+
+watch(
+  () => props.selection,
+  async (value, oldValue) => {
+    errorMessage.value = ``;
+    if (value !== oldValue) {
+      selectedNode.value = undefined;
+      ManualselectedNode.value = undefined;
+    }
   },
   { deep: false },
 );
 
+watch(
+  () => ManualselectedNode.value,
+  async (value, oldValue) => {
+    if (value != undefined || value != null) {
+      selectedNode.value = { nodeId: Number(value) };
```

Comment:
Playground build fails because of this line.
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/40770501/65688699-d396-4e22-ba49-03b0b86bb388)